pkg/app/mutate_app.go

```diff
@@ -258,8 +257,6 @@ func (m *Mutator) mutateLabels(ctx context.Context, app v1alpha1.App, appVersion
 func (m *Mutator) mutateManagementClusterApp(ctx context.Context, oldApp, app v1alpha1.App, operation admissionv1.Operation, appVersionLabel string) ([]mutator.PatchOperation, error) {
```

Comment:
We can remove `mutateManagementClusterApp` func instead. Those apps should not be reconciled by config-controller anymore since your release-operator release.

Reply:
Scratch that we need to update `opsctl` frist